### ISSUE TITLE
Document AWS resources that are not in the CFN Stack

### ISSUE THREAD

**Issue by philoserf** [joejulian/container-aws-createmachines#4]
The EC2 key pair is not part of the CFN stack, and so, is not destroyed when the stack is destroyed.

It and any other resources created should be documented so that they can be cleaned up when the stack is destroyed.

**Comment by joejulian** [comment]
The one time you _don't_ submit a PR when you spot these kinds of issues. lol. This is an easy merge. :)

**Comment by philoserf** [comment]
Yeah. Didn't know if there were more AND I was moving fast building and destroying.

**Comment by joejulian** [comment]
Actually, I'm going to reopen this. I agree with this issue and hope to get to it next week.
